src/android/com/telerik/feedback/AppFeedback.java

```diff
@@ -63,9 +83,49 @@ private void initialize(JSONArray args) throws JSONException {
             this.radFeedback = RadFeedback.instance();
             this.radFeedback.setOnFeedbackFinishedListener(this);
             this.radFeedback.init(apiKey, apiBaseUrl, uid);
+
+            // Change the default BitmapResolver if the app uses Crosswalk
+            if (isCrosswalk) {
+                this.radFeedback.setBitmapResolver(new BitmapResolverCrosswalk());
+            }
+        }
+    }
+
+    private class BitmapResolverCrosswalk implements BitmapResolver {
+        @Override
+        public Bitmap getBitmapFromView(View view) {
+            AppFeedback.this.crosswalkLatch = new CountDownLatch(1);
+            webView.getPluginManager().postMessage("captureXWalkBitmap", this);
+            try {
+                AppFeedback.this.crosswalkLatch.await(5L, TimeUnit.SECONDS);
+            } catch (Exception e) {
+                AlertDialog.Builder latchErrorBuilder = new AlertDialog.Builder(AppFeedback.this.cordova.getActivity());
+                latchErrorBuilder.setTitle("AppFeedback Error");
+                latchErrorBuilder.setMessage("" + e);
+                latchErrorBuilder.setCancelable(false);
+                latchErrorBuilder.setNegativeButton("Cancel", new DialogInterface.OnClickListener() {
```

Comment:
Because from my tests, setCancelable allows the alert to be canceled via press otside the dialog or the back button. I don't want this, instead I want to cancel the alert only via the Cancel button.
https://developer.android.com/reference/android/app/AlertDialog.Builder.html